packages/headless/src/controllers/commerce/core/breadcrumb-manager/headless-core-breadcrumb-manager.ssr.ts

```diff
@@ -0,0 +1,32 @@
+import {ensureAtLeastOneSolutionType} from '../../../../app/commerce-ssr-engine/common.js';
+import {
+  ControllerDefinitionOption,
+  SolutionType,
+  SubControllerDefinitionWithoutProps,
+} from '../../../../app/commerce-ssr-engine/types/common.js';
+import {buildProductListing} from '../../product-listing/headless-product-listing.js';
+import {buildSearch} from '../../search/headless-search.js';
+import {
+  BreadcrumbManager,
+  BreadcrumbManagerState,
+} from './headless-core-breadcrumb-manager.js';
+
+export type {BreadcrumbManager, BreadcrumbManagerState};
+
+/**
+ * Defines a `BreadcrumbManager` controller instance.
+ *
+ * @returns The `BreadcrumbManager` controller definition.
```

Comment:
Should also be marked @internal for now

Reply:
ill put it back for now but in the typedoc PR. I removed all those @internal tags. ssr-commerce is documented in our docs site..
https://docs.coveo.com/en/headless/latest/usage/

Reply:
Ah ok, I'll remove mentions of ssr-commerce from docs for now :loading: 👍